### ISSUE TITLE
Long, single-line pfm_title can cause pfm_platform to not appear

### ISSUE THREAD

**Issue by apizz** [erikberglund/ProfileCreator#119]
It appears in situations where a `pfm_title` is long, but doesn't go over one line, the `pfm_platform` won't appear.  However, if it is longer than two lines it will display.

![screen shot 2019-02-12 at 8 42 09 pm](https://user-images.githubusercontent.com/11789931/52680595-019f3b80-2f07-11e9-973d-80f05287f560.png)

Example for first highlighted preference in above image:
```
<dict>
	<key>pfm_default</key>
	<true/>
	<key>pfm_description</key>
	<string></string>
	<key>pfm_description_reference</key>
	<string>Optional. If false, documents in managed apps and accounts only open in other managed apps and accounts. Default is true. Availability: Available only in iOS 7.0 and later.</string>
	<key>pfm_ios_min</key>
	<string>7.0</string>
	<key>pfm_name</key>
	<string>allowOpenFromManagedToUnmanaged</string>
	<key>pfm_platforms</key>
	<array>
		<string>iOS</string>
	</array>
	<key>pfm_title</key>
	<string>Allow documents from managed sources in unmanaged destinations</string>
	<key>pfm_type</key>
	<string>boolean</string>
</dict>
```

**Comment by apizz** [comment]
Appears to be sliding through and not wrapping:
![screen shot 2019-02-12 at 9 00 45 pm](https://user-images.githubusercontent.com/11789931/52681296-5b086a00-2f09-11e9-8aac-ea622ecbce38.png)

**Comment by apizz** [comment]
Resolved in latest 0.2.5 beta build

**Comment by apizz** [comment]
Fixed in 0.2.5 release
